docs/manual-CN/1.overview/2.quick-start/2.FAQ.md

```diff
@@ -198,7 +161,7 @@ Heartbeat failed, status:RPC failure in MetaClient: N6apache6thrift9transport19T
 
 # meta 日志
 Log line format: [IWEF]mmdd hh:mm:ss.uuuuuu threadid file:line] msg
-E0415 22:32:38.944437 15532 AsyncServerSocket.cpp:762] failed to set SO_REUSEPORT on async server socket Protocol not available
+E0415 22:32:38.944437 15532 AsyncServerSocket.cpp:762] failed to set `SO_REUSEPORT` on async server socket Protocol not available
```

Comment:
I don't think a backquote is necessary here. This is how the log actually looks

Reply:
I prefer to highlight the keyword.
But it's ok to both